src/main/java/net/minecraftforge/event/TickEvent.java

```diff
@@ -35,11 +36,13 @@ public TickEvent(Type type, LogicalSide side, Phase phase)
 
     public static class ServerTickEvent extends TickEvent {
         private final BooleanSupplier haveTime;
+        private MinecraftServer server;
```

Comment:
This field should be `final`, given that nothing else modifies it outside of the constructor.
```suggestion
        private final MinecraftServer server;
```